go/saltpack/packets.go

```diff
@@ -28,7 +28,7 @@ type encryptionHeader struct {
 	Type            MessageType    `codec:"type"`
 	Ephemeral       []byte         `codec:"ephemeral"`
 	SenderSecretbox []byte         `codec:"sendersecretbox"`
-	Receivers       []receiverKeys `codec:"rcvrs"`
+	Receivers       []receiverKeys `codec:"rcvrs,omitempty"`
```

Comment:
Ok, cool, when I inspect a message with anonymous recipients, I do indeed see `nil` in the `ReceiverKID` fields. @maxtaco is that just how our MessagePack codec treats empty slices?